src/redux/reducers/profile.js

```diff
@@ -0,0 +1,39 @@
+import {
+  GET_PROFILE,
+  PROFILE_ERROR,
+  UPDATE_PROFILE_FAIL,
+  UPDATE_PROFILE_SUCCESS,
+} from '../actions/actionTypes/profile';
+
+const initialState = {
+  profile: null,
+  loading: true,
+  error: null,
+};
+
+export default function (state = initialState, action) {
+  const { type, payload } = action;
+
+  switch (type) {
+    case GET_PROFILE:
```

Comment:
By doing this Case statement without expecting to return something is not inadequate. Its like creating an if staement like this;
```
if (GET_PROFILE){

} 
Refactor this in a way that if you expect a case, it should return something like you did in the case block after 
```

Reply:
> By doing this Case statement without expecting to return something is not inadequate. Its like creating an if staement like this;
> 
> ```
> if (GET_PROFILE){
> 
> } 
> Refactor this in a way that if you expect a case, it should return something like you did in the case block after 
> ```

Thanks! I am going to work on that.